datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SecurityAccessManifester.java

```diff
@@ -26,8 +26,7 @@
     public void overrideSecurityAccess(InstanceGroupType instanceGroupType, List<InstanceGroupV4Request> instanceGroups, String securityGroupId, String cidrs) {
         instanceGroups.stream()
                 .filter(ig -> ig.getType() == instanceGroupType)
-                .findFirst()
```

Comment:
@doktoric What was the original intention of applying this only on the first node for a given instance type?

Reply:
if was not the first node but the instancegroup where the type equals

Reply:
Ah sorry my bad. What was the original intention of applying this only on the first instancegroup. This assumes that there could be only one set of nodes for a given instancegroup, no?